.gitignore

```diff
@@ -63,3 +63,4 @@ SortedCFDatabase.def
 htmlcov
 .coverage
 /benchmark/scripts/Benchmark_Driverc
+.vscode
```

Comment:
We usually avoid having people's editor-specific files listed here. There are some options to avoid doing that (you could set it globally or locally): https://help.github.com/en/github/using-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file

Reply:
My bad, I actually don't even use it but I added it to .gitignore at some point and it accidentally got pushed. I will take it back out.

Reply:
Resolved in 1d625d8.